.github/workflows/ci.yaml

```diff
@@ -96,22 +95,42 @@ jobs:
           registry: ${{ env.GITHUB_REG }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Login to AliyunCS_BJ
+      - name: Build the image
+        uses: docker/build-push-action@v5
+        with:
+          platforms: linux/amd64
```

Comment:
does arm64 needed here?

Reply:
Building arm64 image is very slow (almost 1 hour, tested in last [commit](https://github.com/koordinator-sh/koordinator/actions/runs/6445605457/job/17499899839)), so we'd better only build arm image when a PR is merged instead of in each commit.